src/webui/api/searchcontroller.cpp

```diff
@@ -72,8 +72,8 @@ namespace
     {
         QJsonArray categoriesInfo
         {QJsonObject {
-            {QLatin1String("id"), "all"},
-            {QLatin1String("name"), SearchPluginManager::categoryFullName("all")}
+            {QLatin1String("id"), u"all"_qs},
```

Comment:
Yes, I wanted to have `QT_NO_CAST_FROM_ASCII` defined first.

Reply:
>I've also noticed places like QString::fromLatin1("some string") in the code that need to be replaced with u"some string"_qs.

This will come later too.